lizard_auth_server/views.py

```diff
@@ -211,6 +231,20 @@ def form_valid(self, form):
                 kwargs={'invitation_pk': inv.pk}))
 
 
+class ConfirmDeletionUserconsentView(DeleteView):
+    model = UserConsent
+    template_name = 'lizard_auth_server/confirm_deletion_userconsent.html'
+    context_object_name = 'userconsent'
+    success_url = reverse_lazy('profile')
+
+    def delete(self, request, *args, **kwargs):
+        self.object = self.get_object()
+        # Users can only delete their own UserConsents
+        if request.user == self.object.user:
+            self.object.delete()
+        return HttpResponseRedirect(self.get_success_url())
```

Comment:
.... en als de user niet de goede user is krijg je geen foutmelding maar de success url.

Ach, is ook wel prima. Zal alleen optreden als iemand aan het hacken is. Maar misschien even een regeltje uitleg erbij.

Reply:
Done.